packages/@aws-cdk/aws-ecs/test/ec2/test.ec2-task-definition.ts

```diff
@@ -1006,8 +1006,8 @@ export = {
         Family: 'Ec2TaskDef',
         Volumes: [{
           Name: 'scratch',
-          EfsVolumeConfiguration: {
-            FileSystemId: 'local',
+          EFSVolumeConfiguration: {
+            FilesystemId: 'local',
```

Comment:
These changes are going to trigger a **replacement** of existing deployed `TaskDefinition` instances. Was this intended?